lib/defaults/defaults.go

```diff
@@ -252,6 +252,12 @@ const (
 	// is accessible
 	ClusterCheckTimeout = 5 * time.Second
 
+	// StatusCollectionTimeout specifies the timeout for collecting gravity status.
+	StatusCollectionTimeout = 1 * time.Minute
+
+	// AuditCollectionTimeout specifies the timeout for collecting audit logs.
+	AuditCollectionTimeout = 1 * time.Minute
```

Comment:
Nit: Looking at where it is being used, "audit collection" is not very accurate name as it's used for the audit log client which is used for submitting audit events. Something like AuditLogClientTimeout would probably be more appropriate name. Also, I think 1 minute for this is too much, I'd make it like 5 seconds or so.